server/server.go

```diff
@@ -380,11 +383,21 @@ func initRBACService(store store.GaiaStore) (rbac.Service, error) {
 		}
 	}
 
-	svc, err := rbac.NewEnforcerSvc(gaia.Cfg.RBACDebug, store.CasbinStore())
+	model, err := rbac.LoadModel()
+	if err != nil {
+		return nil, fmt.Errorf("error loading model: %w", err)
```

Comment:
Please Log the error and return a userfriendly message. The log is enough to check out the rest. And below too.

Reply:
We wrap the error and log in the calling code. I think these errors are only really for us anyway?
We could return a user friendly error in the calling code that is just a blanket "failed to init RBAC". But here, I'd prefer to wrap and then log where we call initRBACService.